README.md

```diff
@@ -91,8 +90,27 @@ Or, you can use Google Container Engine's free trial to get a 3 node cluster.
 
 ### Get and Run Fission
 
-If you're using GKE, use the fission.yaml unmodified.  If you're using
-minikube, change all instances of LoadBalancer services to NodePort.
+#### local and no cloud provider
+
+If you're using minikube or no cloud provider, change all instances of LoadBalancer services to NodePort.
+
+```
+  $ curl -sL http://fission.io/fission.yaml | sed 's/LoadBalancer/NodePort/g' | kubectl create -f -
```

Comment:
I understand your concerns and I think the best way to go is to split up the `fission.yaml` into two (actually three) files -> 
1.) Namespace, Deployments and no external services `fission.yaml`
2.) External Services  for cloud providers `fission-svcs-cloud.yaml`
3.) External Services with NodePort `fission-nodeport.yaml`

With this approach you don't have to modify two files if the tags of the container images are changed. If you are happy with this way I will modify my PR :)